### PR TITLE
[CN-1890] disable deploy options

### PR DESCRIPTION
**Pull request by pjberry16**
NOTE: I'm merging the changes into the branch release-1.29.x so that when other changes are merged into the upstream branch in spinnaker/deck we should just be able to merge them directly into this branch with our changes.

Disables the same components that are usually disabled by github.com/one-thd/thd-deck/disable.sh

A few of the files disabled won't affect our spinnaker such as in /titus/ and /dcos/ but I figured I should fully disable every case that the disable.sh script would just in case.  

The readme changes include all the steps that I used for creating and testing new deck images and I figure will most likely be added to cd-internal-docs in the spinnaker upgrade section.

### CODE REVIEW

File: packages/azure/src/loadBalancer/details/loadBalancerDetail.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>{{loadBal
             Load Balancer Actions <span class="caret"></span>
           </button>
           <ul class="dropdown-menu" uib-dropdown-menu role="menu">
-            <li><a href ng-click="ctrl.editLoadBalancer()">Edit Load Balancer</a></li>
+            <li class="disabled"><a>Edit Load Balancer</a></li>
```
Comment (victor-homedepot) [inline packages/azure/src/loadBalancer/details/loadBalancerDetail.html:30]: I wonder if the methods should just return false instead so the context isn't lost or if we ever want to enable it for some reason?

File: packages/core/src/loadBalancer/CreateLoadBalancerButton.tsx [hunk 2]
```diff
@@ -38,59 +35,11 @@ export class CreateLoadBalancerButton extends React.Component<ICreateLoadBalance
...
-      },
-    );
-  };
-
   public render() {
     if (!this.state.isDisabled) {
       return (
         <div>
-          <button className="btn btn-sm btn-default" onClick={this.createLoadBalancer}>
+          <button className="btn btn-sm btn-default" disabled={true}>
```
Comment (victor-homedepot) [inline packages/core/src/loadBalancer/CreateLoadBalancerButton.tsx:42]: btw for booleans you only need to have the attribute present `disabled` (`={true}` isn't needed).

File: packages/titus/src/serverGroup/details/serverGroupDetails.html [hunk 3]
```diff
@@ -46,7 +46,7 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
             <li><a href ng-if="!serverGroup.isDisabled" ng-click="ctrl.disableServerGroup()">Disable</a></li>
             <li><a href ng-if="serverGroup.isDisabled" ng-click="ctrl.enableServerGroup()">Enable</a></li>
             <li><a href ng-click="ctrl.destroyServerGroup()">Destroy</a></li>
-            <li><a href ng-click="ctrl.cloneServerGroup()">Clone</a></li>
+            <li class="disabled"><a>Clone</a></li>
```
Comment (victor-homedepot) [inline packages/titus/src/serverGroup/details/serverGroupDetails.html:49]: any reason to keep the anchors around these?

**Review by victor-homedepot — APPROVED** [review]
LGTM

**Review by abe21412 — APPROVED** [review]

**Pull request merged**